cognite/client/_api/simulators/logs.py

```diff
@@ -32,8 +32,7 @@ def retrieve(
     ) -> SimulatorLogList | None: ...
 
     def retrieve(self, id: int | Sequence[int]) -> SimulatorLogList | SimulatorLog | None:
```

Comment:
```suggestion
    def retrieve(self, ids: int | Sequence[int]) -> SimulatorLogList | SimulatorLog | None:
```
change to plural here as well, right?